packages/gatsby-cli/README.md

```diff
@@ -16,68 +16,79 @@ package installed locally.
-|       `--prefix-paths`       | Build site with link paths prefixed (set pathPrefix in your config)                                         |
-|        `--no-uglify`         | Build site without uglifying JS bundles (for debugging)                                                     |
-| `--open-tracing-config-file` | Tracer configuration file (open tracing compatible). See https://www.gatsbyjs.org/docs/performance-tracing/ |
+|            Option            | Description                                                                                                 | Default |
+| :--------------------------: | ----------------------------------------------------------------------------------------------------------- | :-----: |
+|       `--prefix-paths`       | Build site with link paths prefixed (set pathPrefix in your config)                                         | `false` |
+|        `--no-uglify`         | Build site without uglifying JS bundles (for debugging)                                                     | `false` |
+| `--open-tracing-config-file` | Tracer configuration file (open tracing compatible). See https://www.gatsbyjs.org/docs/performance-tracing/ |         |
 
-### Serve
+### `serve`
 
 At the root of a Gatsby site run `gatsby serve` to serve the production build of
 the site for testing.
 
 #### Options
 
-|      Option      | Description                                                                     |
-| :--------------: | ------------------------------------------------------------------------------- |
-|  `-H`, `--host`  | Set host. Defaults to localhost                                                 |
-|  `-p`, `--port`  | Set port. Defaults to 8000                                                      |
-|  `-o`, `--open`  | Open the site in your (default) browser for you                                 |
-| `--prefix-paths` | Serve site with link paths prefixed (if built with pathPrefix in your gatsby-config.js). |
+|      Option      | Description                                                                              |   Default   |
+| :--------------: | ---------------------------------------------------------------------------------------- | :---------: |
+|  `-H`, `--host`  | Set host.                                                                                | `localhost` |
+|  `-p`, `--port`  | Set port.                                                                                |   `8000`    |
+|  `-o`, `--open`  | Open the site in your (default) browser for you                                          |             |
+| `--prefix-paths` | Serve site with link paths prefixed (if built with pathPrefix in your gatsby-config.js). |   `false`   |
 
-### Info
+### `clean`
+
+At the root of a Gatsby site run `gatsby clean` to wipe out `node_modules`, cache, and the `public` directory. This is useful as a last resort when your local project seems to have many issues.
```

Comment:
This would maybe benefit from more of an explanation on _why_ it's a last resort. Are there other things people should try first? Where can they go for more info? How can they help improve the cache so this isn't necessary?

I don't know if we have answers, but it'd be great to include whatever we can here.

Reply:
Great suggestion!